docs/content/latest/api/ysql/commands/ddl_create_table.md

```diff
@@ -80,32 +80,39 @@ postgres=# CREATE TABLE sample(k1 int,
 Table with check constraint
 
 ```sql
-postgres=# CREATE TABLE student_grade (student_id int, 
```

Comment:
Is there a reason for this format change?  You may be able to justify getting rid of the column of whitespace, but I don't think you should add a space after each comma.

Reply:
I've checked several pages with example. On wast majority of them there is no space between table name and bracket in `CREATE TABLE` statement. I've removed that space.